plugins/privacy/user/user.php

```diff
@@ -149,6 +149,30 @@ public function onPrivacyRemoveData(PrivacyTableRequest $request, JUser $user =
 				->delete($this->db->quoteName('#__session'))
 				->where($this->db->quoteName('session_id') . ' IN (' . implode(', ', $quotedIds) . ')')
 		)->execute();
+
+				// Remove sensible user custom field data
+		$query = $this->db->getQuery(true);
+		$query->select($this->db->quoteName(array('params', 'field_id', 'item_id')))
+			->from($this->db->quoteName('#__fields'))
+			->join('INNER', $this->db->quoteName('#__fields_values'), $this->db->quoteName('id') . ' = ' . $this->db->quoteName('field_id'))
+			->where($this->db->quoteName('context') . ' = ' . $this->db->quote('com_users.user'))
+			->where($this->db->quoteName('item_id') . ' =' . $this->db->quote($user->id));
+		
+		$items = $this->db->setQuery($query)->loadAssocList();
+
+		foreach ($items as $item)
+		{
+			$param = json_decode($item['params'], true);
+			if ($param['privacy_related'])
```

Comment:
Please add a clean line before the if statment.

```suggestion

			if ($param['privacy_related'])
```